index.html

```diff
@@ -4,9 +4,52 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Hello, YOUR_NAME</title>
+  <link rel="stylesheet" href="./index_css.css">
+  <title>Hello, my name is Christos Koumpotis</title>
 </head>
 <body>
-  <!-- use this space to add your own content -->
+  <h1> Hello, my name is Christos and this is me : </h1>
+  <img src='./serious.jfif'>
+
+<h1> About me: </h1>
+<h2>I am a freshman at Davidson College </h2>
```

Comment:
Try to use HTML elements to describe the logical structure of your document. Give elements class names + ids and use CSS to change how they appear.

The `<h1>`, `<h2>`, etc. tags are meant to represent header levels. So `<h1>` is a top-level header, `<h2>` is maybe a section heading, `<h3>` a sub-section heading, and so on.

Generic body text should probably be marked up as `<p>...</p>`. If you want a collection of paragraphs to stand out, you can do something like:

```html
<section id="about-me">
  <h2>About Me</h2>
  <p>The reason I am...</p>
  <p>Secret time: ...</p>
</div>
```

And then in CSS:

```css
#about-me p {
  font-size: 20px; /* or whatever */
}
```

In this way, you're saying, "All the `<p>` tags in the `#about-me` section should have a font size of 20px." The HTML only describes the logical structure of the document, though.